packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -399,6 +399,15 @@ class EditableText extends StatefulWidget {
   /// Defaults to EdgeInserts.all(20.0).
   final EdgeInsets scrollPadding;
 
+  static bool get debugDeterministicCursor => _debugDeterministicCursor;
+  static bool _debugDeterministicCursor = false;
+  static set debugDeterministicCursor(bool value) {
```

Comment:
Looks like this setter doesn't really do anything? Could we just use a regular property here?

Reply:
Done.